buildenv/aix/jdk8/yum_install_aix-ppc64.txt

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2017, 2017 IBM Corp. and others
```

Comment:
I notice this file is the same as the jdk9 version (except for the V8 in the comment). Should we create a common directory like jdk8or9 to contain the common source and avoid having to update multiple files if something changes?

Reply:
Prefer to keep them separate as we do for docker files.